csunplugged/tests/classic/loaders/test_classicpages_loader.py

```diff
@@ -0,0 +1,84 @@
+from tests.BaseTestWithDB import BaseTestWithDB
+from classic.models import ClassicPage
+from classic.management.commands._ClassicPagesLoader import ClassicPagesLoader
+from utils.errors.CouldNotFindYAMLFileError import CouldNotFindYAMLFileError
+from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
+from utils.errors.EmptyYAMLFileError import EmptyYAMLFileError
+
+
+class ClassicPageLoaderTest(BaseTestWithDB):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.loader_name = "classicpage"
+        self.BASE_PATH = "tests/classic/loaders/assets/classicpages/"
+
+    def test_basic_classicpage_loader_config(self):
+        config_file = "basic-config.yaml"
+        classicpage_loader = ClassicPagesLoader(
+            structure_dir="",
+            structure_filename=config_file,
+            base_path=self.BASE_PATH,
+        )
+        classicpage_loader.load()
+        pages = ClassicPage.objects.all()
+        self.assertEqual(1, len(pages))
+        page = pages[0]
+        self.assertEqual("Binary numbers", page.name)
+        self.assertEqual("binary-numbers", page.slug)
+        self.assertEqual("https://classic.csunplugged.org/binary-numbers", page.redirect)
```

Comment:
is there no way to shorten this, e.g. `assertQuerysetEqual` or something of the like?

Reply:
Will change.